components/locid_transform/src/expander.rs

```diff
@@ -181,6 +180,37 @@ fn update_langid(
     }
 }
 
+#[inline]
+fn update_langid_minimize(
```

Comment:
I'd revert order - `langid` as a first arg, and the replacements after, but it's a non blocking suggestion.

Reply:
The arguments are in the same order as

```rust
fn update_langid(
    language: Language,
    script: Option<Script>,
    region: Option<Region>,
    langid: &mut LanguageIdentifier,
) -> TransformResult
```